Week2/HW2/4.givingVariables.js

```diff
@@ -0,0 +1,9 @@
+// giving two different variables to y.
+
+let y = 'Í love spring.';
+console.log("The value of my variable y will be: a string");
+console.log (y);
+
+let x = 'Flowers are beautiful.';
+console.log("the value of my variable y will be: a string");
```

Comment:
And will this be the value? I know the `type` will be a string, but what is the value?